src/util/dynamicConfigParser.ts

```diff
@@ -14,7 +14,7 @@ export class DynamicConfigParser {
     //  " {{message.traits.key  ||   \"email\" }} "
     //  " {{ message.traits.key || 1233 }} "
     const defFormat =
-      /^\s*\{\{\s*(?<path>[a-zA-Z_]([a-zA-Z0-9_]*\.[a-zA-Z_][a-zA-Z0-9_]*)+)+\s*\|\|\s*(?<defaultVal>.*)\s*\}\}\s*$/;
+      /^\s*{{\s*(?<path>[A-Z_a-z](\w*\.[A-Z_a-z]\w*)+)+\s*\|\|\s*(?<defaultVal>.*)\s*}}\s*$/;
```

Comment:
what's this change to do with EsLint ?

Reply:
ESLint replaced the redundant parts of the regex with the simplified version.